src/App.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import Homepage from './Components/Homepage.js';
+
+export const App = () =>
+    <>
```

Comment:
I'm not sure if you need this fragment here

Reply:
fixed